hack/new-findings.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
Nit: `check_kubernetes.sh` or `verify_kubernetes.sh` would be a more accurate name.

Reply:
switched to verify-kubernetes.sh